README.md

```diff
@@ -14,7 +14,7 @@
 
 # json-minimizer-webpack-plugin
 
-This plugin uses [JSON.stringify()](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) to minify your JSON.
+This plugin uses [JSON.stringify()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) to minify your JSON.
```

Comment:
Seems like these were links to Russian translation of stringify so changed it to english as well